src/java.base/share/classes/java/lang/reflect/ProxyGenerator.java

```diff
@@ -520,7 +527,7 @@ private void addProxyMethod(Method m, Class<?> fromClass) {
             }
         }
         sigmethods.add(new ProxyMethod(m, sig, m.getSharedParameterTypes(), returnType,
-                exceptionTypes, fromClass));
+                exceptionTypes, fromClass, "m" + proxyMethodCount++));
```

Comment:
This file should be identical to ecdd7f98743ada0e70f1823f5f78f47973c86f75 (it won't compile as is because, for example, `proxyMethodCount` is not declared).

Reply:
Use the upstream version.